src/components/Properties.astro

```diff
@@ -1,8 +1,59 @@
-<div class="my-6">
+---
+const { heading = 'Attributes' } = Astro.props;
+---
+
+<div data-attributes class="my-6">
+  <div class="flex justify-between items-center not-prose py-2">
+    <h3 class="type-subtitle-1">{ heading }</h3>
+    <button data-expand-button class="text-content-tertiary type-overline hover:text-content-secondary">
```

Comment:
@MeganSteenkamp I added a hover state to the button - Jesters weren't keen on underline